src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -262,6 +263,7 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
     CPU_REGISTER_PASS_ARM(manager, ConvertConv1D);
     CPU_REGISTER_PASS_ARM(manager, ConvertGroupConv1D);
     CPU_REGISTER_PASS_ARM(manager, ConvertGroupConvolution);
+    CPU_REGISTER_PASS_ARM(manager, ConvertI32Div);
```

Comment:
Lets put the explanation here: `The plugin computes Divide in floating point precision. To preserve correct math for integer division we need to insert explicit Floor operation.`